awesomplete.css

```diff
@@ -1,4 +1,14 @@
 [hidden] { display: none; }
+[hiddena11y] {
```

Comment:
Good point! I was trying to align to the existing conventions as much as possible. I guess I went too far in this case. I like a class much better for that as well :-)